testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/RefreshTokenTest.java

```diff
@@ -495,6 +499,71 @@ public void refreshTokenReuseTokenWithoutRefreshTokensRevoked() throws Exception
         events.expectRefresh(refreshToken1.getId(), sessionId).assertEvent();
     }
 
+
+    @Test
+    public void refreshTokenReuseTokenWithoutRefreshTokensRevokedWithLessScopes() throws Exception {
+        //add phone,address as optional scope and request them
+        ClientScopeRepresentation phoneScope = adminClient.realm("test").clientScopes().findAll().stream().filter((ClientScopeRepresentation clientScope) ->"phone".equals(clientScope.getName())).findFirst().get();
+        ClientScopeRepresentation addressScope = adminClient.realm("test").clientScopes().findAll().stream().filter((ClientScopeRepresentation clientScope) ->"address".equals(clientScope.getName())).findFirst().get();
+        ClientManager.realm(adminClient.realm("test")).clientId(oauth.getClientId()).addClientScope(phoneScope.getId(),false);
+        ClientManager.realm(adminClient.realm("test")).clientId(oauth.getClientId()).addClientScope(addressScope.getId(),false);
+
+        try {
+
+            //scope parameter consists less optional scope that scope refresh token => error thrown
```

Comment:
Minor (Sorry I overlooked it before): but it might be possible to update this comment a bit as the error is not thrown in this test.

Reply:
You are right. This comment must not be here. I have removed it.

Reply:
Thanks